File: Calculator.java
import java.lang.*;
import java.util.Scanner;
public class Calculator{
    public static void main(String[] args){
        Scanner input = new Scanner(System.in);
        float n1,n2;
        System.out.println("Enter number 1:");
        n1=input.nextFloat();
        System.out.println("Enter number 2:");
        n2=input.nextFloat();
        int choice;
        System.out.println("1.Addition,2.Substraction,3.Division,4.multiplication");
        System.out.println("Enter your choice : ");
        choice=input.nextInt();
        if(choice==1){
            System.out.println(n1+"+"+n2+"="+(n1+n2));
        }
        else if(choice==2){
             System.out.println(n1+"-"+n2+"="+(n1-n2));
        }
        else if(choice==3){
            if(n2!=0)
            {
               System.out.println(n1+"/"+n2+"="+(n1/n2));
            }
            else
            {
                System.out.println("Can't divide by zero");
            }
        }
        else if(choice==4){
             System.out.println(n1+"*"+n2+"="+(n1*n2));
        }
        else{
             System.out.println("Please enter correct choice");
        }
    }
}

File: calculator.cpp
#include <iostream>
using namespace std;
int main()
{
    float n1,n2;
    int choice;
    cout<<"Enter two integers : ";
    cin>>n1;
    cin>>n2;
    cout<<"1.Addition\n2.Subtraction\n3.Multiplication\4.Division\n";
    cout<<"Enter your choice : ";
    cin>>choice;
    switch(choice)
    {
        case 1:
             cout<<"Addition of "<<n1<<"and "<<n2<<" is "<<n1+n2;
             break;
        case 2:
             cout<<"Subtraction of "<<n1<<"and "<<n2<<" is "<<n1-n2;
             break;
        case 3:
             cout<<"Multiplication of "<<n1<<"and "<<n2<<" is "<<n1*n2;
             break;
        case 4:
             if(n2==0){
                 cout<<"Addition is not possible";
             }else{
                cout<<"Division of "<<n1<<"and "<<n2<<" is "<<n1/n2;
                break;
             }
        default:"Choose choice from given options";
    }
}
